### DISCUSSION TITLE
نقشه‌ی آنلاین موقعیت بیسیم‌ها

### DISCUSSION THREAD

**Discussion by MahsaProxy** [WeAreMahsaAmini/Core#483] [Ideas]
ایده‌ی خیلی خام، نمیدونم چقدر امکان داره عملی بشه.

بیسیم‌هایی که دست مامورهاست تا شعاع تقریبا ۵کیلومتری توی رنج فرکانسی uvf سینگال انتقال میدن.
درسته که سیگنالا رمزنگاری شدند ولی منبع سینگال رو میشه تشخیص داد.

اگه بشه بیشتر از ۳تا نقطه‌ی مختلف شهر آنتن بذاریم با قابلیت مانیتور این فرکانس‌ها عملا می‌تونیم موقعیت این این سیگنال‌ها رو توی سطح شهر مشخص کنیم. اگه آنتن‌ها رو داشته باشیم می‌تونیم با درست کردن هیت‌مپ روی نقشه‌ی شهر‌ها تقریبا موقعیت دسته‌های مامورها رو روی نقشه به صورت زنده داشته باشیم و مثلا توی یک وبسایت/کانال تلگرام اطلاع رسانی کنیم.

- نمیدونم چقدر امکانش هست ولی شاید بشه با گیر آوردن بیسیم رمزی روش هست رو برداشت و برای دیکود بقیه استفاده کرد. یا یه جا کلا اینا رو آرشیو کرد که شاید بعدا بشه رمزشون رو شکوند.
- نمیدنم چقدر امکانش هست ولی شاید بشه آموزش ساخت آنتن رو بذاریم و ملت درست کنن تا سطح شهر رو پوشش بدیم.

**Comment by IJustWantSomeFreedom** [comment]
نمونه ای چیزی دارید خودتون که اینکارو کنه؟ من یه سرچ ریز کردم و چیزی پیدا نکردم که همچین چیزی شدنی باشه...

**Reply by MahsaProxy** [reply]
مثال:
https://en.wikipedia.org/wiki/Direction_finding

**Reply by IJustWantSomeFreedom** [reply]
@MahsaProxy 
درسته. متشکر. 
من اطلاعات تخصصی تو این زمینه ندارم. ولی اگر نیاز به فرانت کار داشتید، بنده در خدمتم
